platform.json

```diff
@@ -56,17 +60,49 @@
       "owner": "platformio",
       "version": "~6.51506.0"
     },
+    "framework-zephyr": {
+      "type": "framework",
+      "optional": true,
+      "owner": "platformio",
+      "version": "~2.20700.0"
+    },
     "tool-teensy": {
       "type": "uploader",
       "optional": true,
       "owner": "platformio",
       "version": "<2"
     },
+    "tool-pyocd": {
+      "type": "debugger",
+      "optional": true,
+      "owner": "platformio",
+      "version": "~1.2900.0"
+    },
```

Comment:
How the `tool-pyocd` package is used here?

Reply:
Whoops, I don't think that should have slipped in there, that probably came from when I was copying over the [NXP i.MX RT packages](https://github.com/platformio/platform-nxpimxrt/blob/9e6009659482dd37927bb6e0297c710fc8ef9582/platform.json#L59) and never got cleaned up. I'm pretty sure it can safely be removed.